source/extensions/filters/http/grpc_http1_reverse_bridge/filter.h

```diff
@@ -40,6 +41,17 @@ class Filter : public Envoy::Http::PassThroughFilter {
   // buffer we instead maintain our own.
   Buffer::OwnedImpl buffer_{};
 };
+class FilterConfigPerRoute : public Router::RouteSpecificFilterConfig {
```

Comment:
formatting nit: please add a blank line before and after this class.

Reply:
Fixed in https://github.com/envoyproxy/envoy/pull/8645/commits/0edac0420944a19891d43804acbe6fd3f0edf654